src/Components/CreateImageWizard/steps/ImageOutput/ReleaseSelect.tsx

```diff
@@ -26,6 +26,7 @@ import {
 } from '../../../../store/wizardSlice';
 import isRhel from '../../../../Utilities/isRhel';
 import { toMonthAndYear } from '../../../../Utilities/time';
+import { useFlag } from '../../../Utilities/useGetEnvironment';
```

Comment:
One more level looks like

```suggestion
import { useFlag } from '../../../../Utilities/useGetEnvironment';
```

Reply:
:facepalm: